config/spamd/spamd_db.php

```diff
@@ -122,35 +119,38 @@
 }
 
 /* spam trap e-mail address */
-if($_GET['spamtrapemail'] <> "") {
```

Comment:
Good spotting, to be honest i don't know. As you can see it was already done like that also before i touched the code. But i think your right, it should only be done once.. At least after the edits it worked better than before.. Still fundamentally several things wrong with this package though, but that's a different subject.